### ISSUE TITLE
Change message for valid purchase requests

### ISSUE THREAD

**Issue by tfwoodroofMCS** [CredComSoc/CreditCoopTech#92]
As observed in final bullet point of https://github.com/CredComSoc/CreditCoopTech/issues/35#issuecomment-1691577740, if a buyer makes a purchase request that would not violate either their or the seller's credit limits, the buyer gets an message that reads 'Unable to complete the transaction'. However, it's entirely possible to complete the transaction workflow.

Change this message to 'Thank you for your purchase request. The seller has been notified, and you will receive a notification as soon as they have either approved or declined the request.'

**Comment by tfwoodroofMCS** [comment]
@yonasbek - texts on both header and main message as implemented are different (and slightly less informative) than the in the issue. If you could use the exact ones above, that would be great.

**Comment by yonasbek** [comment]
@tfwoodroofMCS I wanted to raise this question tomorrow but lets try to figure this out. So if the user adds 5 items to cart from 3 different sellers then which seller name should i put there when the complete purchase is a success? The other question would be if from 2 seller the transaction was successful and from one of the sellers the transaction fails what would be the result? Because this one is tricky because if one fails we don't know which one because we create the transaction in bulk(suggestion create another button for each items then create transaction and send notification other wise it gets tricky).

**Comment by tfwoodroofMCS** [comment]
Very good questions...

For the first one (successful purchase requests to multiple sellers), then the main message should read 'Thank you for your purchase request. The seller(s) have been notified, and you will receive a notification as soon as they have either approved or declined the request.' (i.e. the same as initially stated, but with any seller username). This can also be the case if there is just one seller. So any entirely valid purchase request should get this response.

For the second, I think the simplest thing for now is that if the total price of all items (from multiple sellers or not) is greater than the user's available credit, then they should receive the 'Insufficient balance' message in #91. We can work out something more sophisticated (e.g. progress some subset of the items that would keep the user within their credit limit) later on.

**Comment by yonasbek** [comment]
![Image](https://github.com/CredComSoc/CreditCoopTech/assets/49316951/3876fc07-357a-440d-b8ed-fd66b820ad7b)

This is the current success message should we change this?

**Comment by tfwoodroofMCS** [comment]
Yes please - I've written the texts very carefully to give the users as much information as possible whilst minimising ambiguity.

**Comment by dilgreen** [comment]
Treating transactions in bulk is a bad idea, for many reasons, simply stated as 'exposing ourselves to multiple possible situations with complicated differences.'.

We should loop through cart items one by one, in the order added.

We have many issues to address, and this one should be kept simple for now.

Most transactions on these systems are single item transactions.

**Comment by tfwoodroofMCS** [comment]
@dilgreen - if you look at the functionality as it is now, it doesn't have any affordances for treating transactions in anything other than bulk. The point of the fixes I've suggested here (and in related issues) is to keep things simple and robust, whilst still giving the user some obvious pointers as to what to do.

As you say, there are loads of issues to address, and I can't see how adding loops here and elsewhere would be simpler than what I've suggested. However, I've already created issue #116 to discuss options.
